releasenotes/notes/upgrade-curve-fit-4dc01b1db55ee398.yaml

```
@@ -0,0 +1,9 @@
+---
+others:
+  - |
+    Curve analysis is updated to use the covariance between fit parameters in the error propagation.
```

This should be under `upgrade` section and include API doc links
```suggestion
    The :class:`CurveAnalysis` class has been updated to use the covariance between fit parameters in the error propagation.
```
* There should be another upgrade bullet saying that the analysis results of specific experiments (probably list them with API links) has been changed from FitVal to UFloat.
*  Also there should be a depreciation section saying that FitVal has been deprecated